### ISSUE TITLE
create step-1 form for "User Group" Module

### ISSUE THREAD

**Issue by muhammadawaisshaikh** [muhammadawaisshaikh/community-events-angular#29]
![image](https://user-images.githubusercontent.com/24633059/135679681-d8d9bdd0-3dcc-4fee-800c-ff71cc4880ce.png)

- develop this simple form at step-1 tab. 
- Refer this Issue for Stepper (https://github.com/muhammadawaisshaikh/community-events-angular/issues/28)

**Comment by GregSithole** [comment]
Hey, please can you assign this to me? I'll give it a go

**Comment by muhammadawaisshaikh** [comment]
i will love to assign this to you. but this is interconnected with #27 #28

**Comment by muhammadawaisshaikh** [comment]
what specifically your skillset is? so that i can suggest you the issue to takeover?

**Comment by muhammadawaisshaikh** [comment]
are you familiar with NodeJS? i am also planning to maintain a repo of the Backend for this project,

**Comment by muhammadawaisshaikh** [comment]
Don't forget to hit the **Star** button for this repo. please it's a must todo thing before working on any issue.

**Comment by GregSithole** [comment]
@muhammadawaisshaikh Yes I'm very familar with NodeJS and Angular, I've been working with those for the last 3-4 Years so I think I can manage this section. I could do the inter-connected issues but I also don't wanna take away from other people's particpation with the whole hacktoberfest happening

**Comment by muhammadawaisshaikh** [comment]
don't worry about it, you can work on #27 and #28 if you want, i will love to assign both to you from this repo. waiting for your response.

**Comment by GregSithole** [comment]
Yes that's perfectly fine with me thanks 👌🏾

**Comment by muhammadawaisshaikh** [comment]
so the dependant work for the tickets is in development, i will assign 6 small tickets to you related to small forms and steps to be developed there. #29 #30 #31 #32 #33 #34 i will assign these to you once dependant work is finished by other guy.

**Comment by GregSithole** [comment]
Perfectly happy with that, and thanks

**Comment by muhammadawaisshaikh** [comment]
Hey, i am happy to assign you #27 #28 and #29, please comment on these 3 issues so that i can able to assign those 3 to you, that guy who were working on them is not interested to do the work, i thought you will be best fit for these 3. thanks

**Comment by GregSithole** [comment]
Awesome, and yes please do assign and thanks again man!

**Comment by muhammadawaisshaikh** [comment]
any update?
